code/game/objects/items/weapons/tanks/tank_types.dm

```diff
@@ -113,9 +113,12 @@
  */
 
 /obj/item/weapon/tank/internals/plasmaman
+	name = "plasmaman plasma tank"
+	desc = "A tank of plasma gas."
 	icon_state = "plasmaman_tank"
 	item_state = "plasmaman_tank"
 	force = 10

```

Comment:
That's the force for the big plasmaman tank which fits on the back slot and not the belt. Normal-sized oxygen tanks, anaesthetic tanks and air tanks have 10 force as well.

The plasmaman tank that fits on the belt (and is carried by plasmamen) has 5 force, 1 point greater than the smaller emergency oxygen tanks.